packages/unpublished-test-infra/addon-test-support/qunit-asserts/assert-deprecation.ts

```diff
@@ -156,12 +161,32 @@ export function configureDeprecationHandler() {
       };
     }
 
+    if (config.when) {
+      let libs = Object.keys(config.when);
+      for (let i = 0; i < libs.length; i++) {
+        let library = libs[i];
+        let version = config.when[library]!;
+        let sanitizedVersion = version.replace(/[\^><=~]/g, '');
+
+        if (version.indexOf('<=') === 0) {
+          if (!lte(library, sanitizedVersion)) {
+            return;
```

Comment:
we need to always execute the callback if it exists, so this isn't quite right.

Reply:
we also need to push a "success" status in case any `assert.expect()` counts were configured for us.